src/runtime/runtime.js

```diff
@@ -816,6 +823,20 @@ export class ConfiguredRuntime {
     this.buttonApi_.attach(button, optionsOrCallback, opt_callback);
   }
 
+  /** @override */
+  attachSmartButton(container, optionsOrCallback, opt_callback) {
+    if (!isExperimentOn(this.win_, ExperimentFlags.SMARTBOX)) {
+      throw new Error('Not yet launched!');
+    }
+    if (!container) {
+      throw new Error('No container element found to render Smart button!');
+    }
+    this.documentParsed_.then(() => {
```

Comment:
I'm actually not sure why we need to wait for `documentParsed` here. Can you explain?

Reply:
That was earlier implementation, when this was defined as a `flow`. Removed.